src/signtool/IBM_HexString.h

```diff
@@ -0,0 +1,194 @@
+/* IBM_PROLOG_BEGIN_TAG                                                   */
+/* This is an automatically generated prolog.                             */
+/*                                                                        */
+/* $Source: src/signtool/IBM_HexString.h $                                */
+/*                                                                        */
+/* OpenPOWER sb-signing-utils Project                                     */
+/*                                                                        */
+/* Contributors Listed Below - COPYRIGHT 2016                             */
+/* [+] International Business Machines Corp.                              */
+/*                                                                        */
+/*                                                                        */
+/* Licensed under the Apache License, Version 2.0 (the "License");        */
+/* you may not use this file except in compliance with the License.       */
+/* You may obtain a copy of the License at                                */
+/*                                                                        */
+/*     http://www.apache.org/licenses/LICENSE-2.0                         */
+/*                                                                        */
+/* Unless required by applicable law or agreed to in writing, software    */
+/* distributed under the License is distributed on an "AS IS" BASIS,      */
+/* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or        */
+/* implied. See the License for the specific language governing           */
+/* permissions and limitations under the License.                         */
+/*                                                                        */
+/* IBM_PROLOG_END_TAG                                                     */
+
+#ifndef __IBM_HEXSTRING_H_
+#define __IBM_HEXSTRING_H_
+
+#include <stdint.h>
+
+#include <string>
+#include <vector>
+#include <istream>
+#include <ostream>
+#include <sstream>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+
+
+//  A hex string object.
+
+typedef unsigned char byte;
+
+typedef std::vector<byte> IBM_HexBytes;
+
+class IBM_HexString
```

Comment:
Why is this some kind of special IBM style of hexstring?

IS it like powerpc bit ordering, in that we have to be all special?